pilot/pkg/model/context.go

```diff
@@ -477,7 +477,7 @@ type BootstrapNodeMetadata struct {
 	Owner string `json:"OWNER,omitempty"`
 
 	// ProxyViaAgent specifies whether xDS streams are proxied through the agent.
-	ProxyViaAgent bool `json:"PROXY_VIA_AGENT,omitempty"`
+	ProxyViaAgent bool `json:"PROXY_VIA_AGENT,string,omitempty"`
```

Comment:
TIL about `string` option...

In other places we use StringBool. We should probably be consistent across the two. Likely this method is better